guide_drafts/model-derives.md

```diff
@@ -0,0 +1,926 @@
+Model Derives in Depth
+===============
+
+Diesel lets you write safe queries by using traits that you can implement on your models.
+Writing these implementations by hand can be very laborious, so in addition to the `diesel` crate, the `diesel_codegen` crate offers custom derives.
+In this section, we will cover each trait in detail in terms of its use cases, usage considerations, as well compare hand written implementations to its derive.
+To make use of these derives, `diesel_codegen` must be imported into the root of your project.
+
+Throughout this guide, we will be looking at examples for each trait and how they interact together.
+Some of the example code will be implementing basic [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) database operations.
+We'll be covering creating, reading, and updating data.
+The details of those operations will be not be covered beyond their relevance to the demonstrated trait.
+
+- [Queryable](#queryable)
+- [Insertable](#insertable)
+- [Identifiable](#identifiable)
+- [AsChangeset](#aschangeset)
+- [Associations](#associations)
+
+## <a name="queryable"></a>Queryable
+
+Using the [`#[derive(Queryable)]`](http://docs.diesel.rs/diesel/query_source/trait.Queryable.html)` trait annotation on your model struct allows records to be queried from the database.
```

Comment:
stray back tick

Also, i'd use ref links everywhere, i.e.

```markdown
Using the [`#[derive(Queryable)]`] trait annotation on your model struct allows records to be queried from the database.

[`#[derive(Queryable)]`]: http://docs.diesel.rs/diesel/query_source/trait.Queryable.html
```

which make this easier to read and catch syntax stuff like that